scheduler/src/cook/mesos.clj

```diff
@@ -52,18 +52,18 @@
      (let [submit-time (java.util.Date.)]
        (doseq [{:keys [uuid command ncpus memory name retry-count priority]} jobs
                :let [txn {:db/id (d/tempid :db.part/user)
-                          :job/uuid uuid
```

Comment:
Do we really want lexicographic ordering here? Or is it more useful to use the pattern of having the uuid first?